pinot-controller/src/main/java/org/apache/pinot/controller/ControllerConf.java

```diff
@@ -240,6 +242,8 @@ private static long getRandomInitialDelayInSeconds() {
   private static final int DEFAULT_JERSEY_ADMIN_PORT = 21000;
   private static final String DEFAULT_ACCESS_CONTROL_FACTORY_CLASS =
       "org.apache.pinot.controller.api.access.AllowAllAccessFactory";
+  private static final String DEFAULT_ACCESS_CONTROL_USERNAME = "admin";
+  private static final String DEFAULT_ACCESS_CONTROL_PASSWORD = "admin";
```

Comment:
```suggestion
  private static final String DEFAULT_ACCESS_CONTROL_PASSWORD = "passwd";
```

Reply:
althought I dont know if we should have a default --> if not configured we should simply throw exception if a secure auth enabled cluster